packages/rfw/test/runtime_test.dart

```diff
@@ -1152,7 +1152,28 @@ void main() {
 
     expect(tester.takeException().toString(), contains(expectedErrorMessage));
     expect(find.byType(ErrorWidget), findsOneWidget);
-    expect(tester.widget<ErrorWidget>(find.byType(ErrorWidget)).message, expectedErrorMessage);
+    expect(tester.widget<ErrorWidget>(find.byType(ErrorWidget)).message, contains(expectedErrorMessage));
+  });
+
+
+  testWidgets('Customized error widget', (WidgetTester tester) async {
+    final ErrorWidgetBuilder oldBuilder = ErrorWidget.builder;
+    ErrorWidget.builder = (FlutterErrorDetails details) {
+      return const Text('oopsie!', textDirection: TextDirection.ltr);
+    };
+    final Runtime runtime = Runtime()
+      ..update(const LibraryName(<String>['a']), parseLibraryFile(''));
+    final DynamicContent data = DynamicContent();
+    await tester.pumpWidget(
+      RemoteWidget(
+        runtime: runtime,
+        data: data,
+        widget: const FullyQualifiedWidgetName(LibraryName(<String>['a']), 'widget'),
+      ),
+    );
+    expect(tester.takeException().toString(), contains('Could not find remote widget named'));
+    expect(find.text('oopsie!'), findsOneWidget);
```

Comment:
should probably also check that you can't find an ErrorWidget.

Reply:
Done!